docs/guides/advanced/schemas.md

```diff
@@ -136,6 +136,70 @@ let MovieSchema = coda.makeObjectSchema({
 By default all properties are considered optional, but you can add `required: true` to the property's schema to indicate that the property is required. This adds some type checking to help ensure that formulas return all the required properties, but it cannot catch all cases.
 
 
+### Property descriptions
+
+While a good property name may speak for itself, it's often useful to add additional context about what kind of information the property contains. This can be done by adding a `description` field to the property schema:
+
+
+```{.ts hl_lines="5 10 14"}
+let PersonSchema = coda.makeObjectSchema({
+  properties: {
+    name: {
+      type: coda.ValueType.String,
+      description: "The full name of the person.",
+    },
+    born: {
+      type: coda.ValueType.String,
+      codaType: coda.ValueHintType.Date,
+      description: "The date when the person was born.",
+    },
+    age: {
+      type: coda.ValueType.Number,
+      description: "The age of the person, in years.",
+    },
+  },
+  // ...
+});
+```
+
+These descriptions are shown in the formula editor when inspecting a property.
+
+<img src="../../../images/schemas_descriptions.png" srcset="../../../images/schemas_descriptions_2x.png 2x" class="screenshot" alt="A schema property description being shown in the formula editor">
+
+If the schema is used in a sync table the descritpions will also be used to populate the initial [column descriptions](#column-descriptions).
```

Comment:
If the schema is used as a column format, when projecting child properties using the column header menu or value dropdown the column description will also be populated.

Reply:
Good callout. I've removed the column descriptions section and instead made a tabbed widget with the different places that property descriptions are displayed. PTAL.